components/ui/AppCard.stories.ts

```diff
@@ -15,15 +15,18 @@ export default {
 const Template: Story = (args, { argTypes }) => ({
   components: { AppCard },
   props: Object.keys(argTypes),
-  template: '<AppCard :cta-label="ctaLabel" :image="image" :tags="tags" :title="title" :to="to">{{ vSlot }}</AppCard>'
+  template: `
+    <AppCard :cta-label="ctaLabel" :image="image" :tags="tags" :title="title" :to="to">
+      {{slotDefault}}
```

Comment:
just curious where `slotDefault` change came from... is it for readability?

Reply:
I think so, @techtolentino

Reply:
@techtolentino , it's an "arg" that gets registered in the component so that we can later change it in the Storybook story dynamically.

This way we can the easily test how the component reacts with no text, some text and a lot of text for example. 🙂